_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_par.cpp

```diff
@@ -1874,9 +1874,11 @@ mfxStatus CheckSurface(
 
     if (video.m_inMemType == INPUT_SYSTEM_MEMORY)
     {
-        MFX_CHECK(surface.Data.Y != 0, MFX_ERR_NULL_PTR);
-        MFX_CHECK(surface.Data.U != 0, MFX_ERR_NULL_PTR);
-        MFX_CHECK(surface.Data.V != 0, MFX_ERR_NULL_PTR);
+        MFX_CHECK(!LumaIsNull(&surface), MFX_ERR_NULL_PTR);
```

Comment:
Traditional .Y is not checked.

Reply:
Fixed

Reply:
Now  MFX_CHECK(surface.Data.Y != 0, MFX_ERR_NULL_PTR); below is not needed. Done in !LumaIsNull().

Reply:
Fixed